src/com/videojs/VideoJSModel.as

```diff
@@ -203,6 +203,26 @@ package com.videojs{
             }
         }
 
+        public function get audioTrack():int {
+            if(_provider) {
+                return _provider.audioTrack;
+            }
+            return 0;
```

Comment:
since the other providers that don't support this return `-1`, I think it would make sense to return `-1` here for the case that we do not have a provider